commands/init.go

```diff
@@ -0,0 +1,102 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// InitCmd creates the necessary files required to deploy an app (server.conf)
+type InitCmd struct {
+	StackName string `optional default:"nodejs-basic" short:"s" help:"Name of stack to deploy. Default is nodejs-basic"`
+}
+
+// Copy and paste of server.conf
+func buildServerConf() []byte {
+	return []byte(
+		`location / {
+	proxy_set_header X-Forwarded-For $http_x_forwarded_for;
+	proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
+	proxy_set_header Host $host;
+	include /etc/nginx/section.module/node.conf;
+}
+		
+location ~ "/notnuxt/" {
+	proxy_set_header X-Forwarded-For $http_x_forwarded_for;
+	proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
+	proxy_set_header Host $host;
+	proxy_pass http://next-hop;
```

Comment:
I don't think this is necessary or useful unless they are using proxies that are higher up in the chain. Most people using sectionctl probably aren't initializing their repo with such an advanced config, but the next hop capability should definitely be documented in our docs.

Reply:
I've included it because it's in the `server.conf` in the demo app. I think it's safe to keep in in the event that they want additional proxies in the future. Is there a reason to remove it?

Reply:
at the very least it shouldn't be called /notnuxt/ for a general application of it

Reply:
So I should remove this entire location part?

```
location ~ "/notnuxt/" {
	proxy_set_header X-Forwarded-For $http_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
	proxy_set_header Host $host;
	proxy_pass http://next-hop;
    }`)
}
```

Reply:
Maybe comment it out to show it's not required, and just be like `/next-proxy-hop/`